core/tests/Test.Microsoft.Identity.Core.Unit/InstanceTests/AdfsAuthorityTests.cs

```diff
@@ -388,9 +287,9 @@ public void FailedTenantDiscoveryMissingEndpointsTest()
                 }).GetAwaiter().GetResult();
                 Assert.Fail("validation should have failed here");
             }
-            catch (TestServiceException exc)
+            catch (Exception exc)
```

Comment:
>Exception [](start = 19, length = 9)

can this be a more specific exception? Things could fail for other reasons. Also the assert seems irrelevant.